react/MuiCozyTheme/ExpansionPanel/theme.js

```diff
@@ -0,0 +1,23 @@
+const radiusValue = '6px'
+const borderRadius = {
+  borderTopLeftRadius: radiusValue,
+  borderTopRightRadius: radiusValue,
+  borderBottomLeftRadius: radiusValue,
+  borderBottomRightRadius: radiusValue
+}
+
+const MuiExpansionPanel = {
+  root: {
+    ...borderRadius,
```

Comment:
Shouldn't this be in the "shape/borderRadius" at the root of the materialUI theme to be the default for us instead of overriding individual components ?

Reply:
Actually I didn't know this `theme.shape.borderRadius` thing. I think you're right.